### PR TITLE
fix(curriculum): add missing word in instructions - step 31

### LINKED ISSUE #44602: Learn HTML Forms by Building a Registration Form - Step 31 - missing word in instructions
**Looking forward for reporting a security issue:**
Please report security issues by sending an email to `security@freecodecamp.org` instead of raising a GitHub issue.

**Describe the bug**
**Re: The BETA Version of the responsive web design course.

In the “Learn HTML Forms by building a registration form” section, in step 31 the prompt asks the user to enter in “freeCodeCamp YouTube” as an option in the drop down. The error message pops up saying I should have entered “freeCodeCamp YouTube *Channel*”. The “channel” portion isn’t in the initial prompt.**

**To Reproduce**
1. Go to Step 31 of the building a registration section
2. follow the given instructions
3. Submit response

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Chrome
 - Version: 96.0.4664.110

**Additional context**

### PR DESCRIPTION
**Pull request by ghost**
Learn HTML Forms by Building a Registration Form - Step 31 - Edited the missing word **channel** in the instructions

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #44602

### CODE REVIEW

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/step-031.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ The dropdown options are currently empty. To give them content, add the followin
 ```md
 (select one)
 freeCodeCamp News
-freeCodeCamp YouTube
+freeCodeCamp Channel YouTube
```
Comment (gikf) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/step-031.md:15]:
```suggestion
freeCodeCamp YouTube Channel
```
I think this was supposed to be the other way around :slightly_smiling_face:

**Review by gikf — CHANGES_REQUESTED** [review]

**Review by gikf — APPROVED** [review]
Looks good :+1:

**Comment by najmieh** [comment]
Yes, I had another language in my head! please excuse that!

**Review by jdwilkin4 — APPROVED** [review]
LGTM! 👍

**Pull request merged**